www/src/components/repository/queries.js

```diff
@@ -37,6 +37,7 @@ export const REPOSITORY_QUERY = gql`
         name: tag
       }
       readme
+      main_branch
```

Comment:
we should probably camel case these actually to match the conventions elsewhere (the gql api supports both patterns)